libwfssl/src/ssl.c

```diff
@@ -231,6 +231,12 @@ int ssl_callback_ServerNameIndication(SSL *ssl, int *al, tcn_ssl_ctxt_t *c)
 #define REQUIRE_SSL_SYMBOL(symb) REQUIRE_SSL_SYMBOL_ALIAS(symb, symb);
 #define REQUIRE_CRYPTO_SYMBOL(symb) REQUIRE_CRYPTO_SYMBOL_ALIAS(symb, symb);
 
+#ifdef __GLIBC__
```

Comment:
You might give a short comment why hardcoded values are needed and don't hurt here. Maybe with a link to the Jira?

Reply:
Done